storage/src/main/java/tech/pegasys/teku/storage/client/RecentChainData.java

```diff
@@ -133,17 +131,31 @@ public Store getStore() {
    * @param slot
    */
   public void updateBestBlock(Bytes32 root, UnsignedLong slot) {
```

Comment:
nit: it might be nicer to use this
```suggestion
  public synchronized void updateBestBlock(Bytes32 root, UnsignedLong slot) {
```

Reply:
I used a synchronized block because I didn't want to end up executing whatever is hanging off of `bestBlockInitialized` in the synchronized section.